src-tauri/src/lib.rs

```diff
@@ -294,9 +297,28 @@ pub fn launch_northstar(
     }
 
     // Only Windows with Steam or Origin are supported at the moment
-    if host_os == "windows"
-        && (matches!(game_install.install_type, InstallType::STEAM)
-            || matches!(game_install.install_type, InstallType::ORIGIN)
+    if matches!(game_install.install_type, InstallType::STEAM)
+    {
+        // Create run_northstar.txt for NorthstarProton
+        if host_os != "windows"
+        {
+            let mut file = std::fs::OpenOptions::new()
+                .create(true)
+                .write(true)
+                .truncate(true)
+                .open("run_northstar.txt").unwrap();
+
+            let mut content: &[u8] = b"1";
+            std::io::copy(&mut content, &mut file).unwrap();
+        }
+
+        return match open::that("steam://run/1237970/") {
```

Comment:
right now no arguments are passed, meaning it uses whatever arguments were set up on Steam.

Should this be kept, so that the launch arguments the user decided on are used, or should `-northstar` be enforced?

Reply:
If you do not use `run_northstar.txt` on Windows, you'd need to pass `-northstar`, otherwise it'll launch vanilla Titanfall2. Not sure if there was a Steam URI that allowed launching with params. If not we might have to use `run_northstar.txt` on Windows as well.

Reply:
You can launch a game with arguments via the browser protocol: `steam://run/<id>//<args>/`

but it gives the user a prompt showing them what arguments were passed
![image](https://user-images.githubusercontent.com/15076013/218285620-03e5d76f-675a-4543-85af-83a267bb02c1.png)

This ruins the UX so I think relying on `run_northstar.txt` is the way to go here.

Reply:
Agreed :D

(Also kinda glad Steam has that check ^^)